components/css/input/input.css

```diff
@@ -0,0 +1,38 @@
+.input-container{
+    width:20%;
+    min-width:300px;
+  }
+.input-container *{
+    margin:12px;
+  }
+.input, .textarea{
+    width:100%;
+    padding:.25rem 12px;
+    border-radius:.3rem;
+    height:1.8rem;
+    outline:none;
+    border:none;
+    
+  }
+.input:hover,.textarea:hover{
+    border:1px solid #D1D5DB;
+    transition:.5s;
+  }
+.input:focus, .textarea:focus{
+    border:none;
+    box-shadow:0px 0px 12px #D1D5DB;
```

Comment:
Suggestion  if you are using same color at many places you can use variable instead of repeating yourself.